src/app/components/hris/employees/employee-profile/employee-profile.component.ts

```diff
@@ -53,6 +54,7 @@ export class EmployeeProfileComponent implements OnChanges {
   employeePhysicalAddress !: EmployeeAddress;
   employeePostalAddress !: EmployeeAddress;
   terminationData !: EmployeeTermination
+  BIGemployeeProfile! : EmployeeProfileNew | null;
```

Comment:
Refactor: is there a better name for this? :) also is this being used?

Reply:
removed